packages/vaadin-login/test/login-form.test.js

```diff
@@ -187,21 +186,21 @@ describe('login form', () => {
     enter(vaadinLoginPassword);
     expect(submitStub.called).to.be.false;
 
-    tap(submit);
+    click(submit);
     expect(submitStub.called).to.be.false;
   });
 
   it('should not disable button on button click if form is invalid', () => {
     const submit = login.querySelector('vaadin-button[part="vaadin-login-submit"]');
     expect(submit.disabled).to.not.be.true;
-    tap(submit);
+    click(submit);
     expect(submit.disabled).to.not.be.true;
   });
 
   it('should disable button on button click if form is valid', () => {
     const submit = login.querySelector('vaadin-button[part="vaadin-login-submit"]');
     fillUsernameAndPassword(login);
-    tap(submit);
+    click(submit);
```

Comment:
question: not clear why a `tap` is replaced by a `click`. was it intentional?

Reply:
Restored.